ethereum-types/Cargo.toml

```diff
@@ -1,22 +1,18 @@
 [package]
 name = "ethereum-types"
-version = "0.3.3"
+version = "0.4.0"
 authors = ["Parity Technologies <admin@parity.io>"]
 license = "MIT"
 homepage = "https://github.com/paritytech/primitives"
 description = "Ethereum types"
-build = "build.rs"
-
-[build-dependencies]
-rustc_version = "0.2"
 
 [dependencies]
-crunchy = "0.1.5"
+crunchy = "0.1"
 ethbloom = { path = "../ethbloom", version = "0.5.0", default-features = false }
 ethereum-types-serialize = { version = "0.2.1", path = "../serialize", optional = true }
 fixed-hash = { git = "https://github.com/paritytech/parity-common", default_features = false }
 serde = { version = "1.0", optional = true }
-uint = { git = "https://github.com/paritytech/parity-common", default_features = false }
+uint = { version = "0.4.1", git = "https://github.com/paritytech/parity-common", default_features = false }
```

Comment:
why not from crates.io?

Reply:
we should use either `version =` or `git =`. Never both of them

Reply:
done